DelvUI/Interface/PartyCooldowns/PartyCooldownsManager.cs

```diff
@@ -167,8 +167,10 @@ private void OnMembersChanged(PartyManager sender)
             else if (!_config.ShowOnlyInDuties || Plugin.Condition[ConditionFlag.BoundByDuty])
             {
                 // add new members
-                foreach (PartyFramesMember member in sender.GroupMembers)
+                foreach (IPartyFramesMember partyFramesMember in sender.GroupMembers)
                 {
+                    PartyFramesMember member = (PartyFramesMember)partyFramesMember;
+
```

Comment:
You shouldn't need to do this, the other change is correct though

Reply:
The reason for that was to make as few changes as possible, as otherwise there's several methods that needs changing (CooldownsForMember etc etc)